pyomo/common/autoslots.py

```diff
@@ -270,9 +270,13 @@ def __deepcopy__(self, memo):
             # Note: this implementation avoids deepcopying the temporary
             # 'state' list, significantly speeding things up.
             memo[id(self)] = ans = self.__class__.__new__(self.__class__)
-            ans.__setstate__(
-                [fast_deepcopy(field, memo) for field in self.__getstate__()]
-            )
+            state = self.__getstate__()
+            ans.__setstate__([fast_deepcopy(field, memo) for field in state])
+            # The state ises a temporary dict to store the (mapped)
```

Comment:
Typo (`ises`) but I'm not sure what the word is supposed to be? Maybe just `is`?